clusterloader2/testing/density/config.yaml

```diff
@@ -72,6 +73,9 @@ steps:
       action: start
       nodeMode: {{$NODE_MODE}}
       resourceConstraints: {{$DENSITY_RESOURCE_CONSTRAINTS_FILE}}
+      {{if $ENABLE_SYSTEM_POD_METRICS}}
+      systemPodMetricsEnabled: true
+      {{end}}
```

Comment:
Just out of the curiosity, why can't we have something like:
```
systemPodMetricsEnabled: {{$ENABLE_SYSTEM_POD_METRICS}}
```

Reply:
The difference is how we will handle ENABLE_SYSTEM_POD_METRICS=false case.

In this PR we won't set systemPodMetricsEnabled at all, in your version we'll set systemPodMetricsEnabled=false.

They are handled in the same way so it should be equivalent. I actually like your version better, will change.